integration/integration_test.go

```diff
@@ -1900,6 +1900,11 @@ func (s *IntSuite) TestDiscovery(c *check.C) {
 	// now disconnect the main proxy and make sure it will reconnect eventually
 	lb.RemoveBackend(mainProxyAddr)
 
+	//
+	// Once the proxy is removed, only one proxy is left. Wait for the now
+	// invalid tunnel connection to TTL and be removed.
+	waitForTunnelConnections(c, main.Process.GetAuthServer(), remote.Secrets.SiteName, 1)
```

Comment:
let's use `waitForActiveTunnelConnections` as it's in memory tracked and I wanted to delete `waitForTunnelConnections` later completely